src/lib/actions/customCode.js

```diff
@@ -111,10 +113,15 @@ module.exports = function(settings, event) {
   if (action.settings.isExternal) {
     return loadCodeSequentially(source).then(function(source) {
       if (source) {
-        postscribeWrite(decorateCode(action, source));
+        decoratedResult = decorateCode(action, source);
```

Comment:
It seems like having `decorateCode` pull `source` from `action.settings.source` would simplify code (both source and tests).

Reply:
Talked on slack about this. Since we have a special case for the external custom code we decided to leave it like it is. https://github.com/adobe/reactor-extension-core/blob/master/src/lib/actions/customCode.js#L112